test/suites/edgexfoundry/main_test.go

```diff
@@ -34,8 +34,7 @@ func TestCommon(t *testing.T) {
 		TestChangePort: utils.ConfigChangePort{
 			App:                      supportSchedulerApp,
 			DefaultPort:              supportSchedulerServicePort,
-			TestLegacyEnvConfig:      false, // schemes differ, run specific test instead
-			TestAppConfig:            true,
+			TestAppConfig:            false, // covered in startup message testing
```

Comment:
Do you mean covered by [TestChangeStartupMsg_mixedGlobalApp](https://github.com/canonical/edgex-snap-testing/blob/2d4c28655de18d9c039d6d54465c5b60a73ebde8/test/suites/edgexfoundry/config_test.go#L42)?

Reply:
I just realized we don't have the app option testing in the suite. The one from utils doesn't work because it assumes services are stopped by default.

Reply:
Added in https://github.com/canonical/edgex-snap-testing/pull/144/commits/022b4201bfa0d38d8ed3fae929a667870d16f96d